luigi_pipeline/lib/hail_tasks.py

```diff
@@ -39,12 +40,21 @@ class HailMatrixTableTask(luigi.Task):
     to provide specific operations.
     Does not run if dest path exists (complete) or the source path does not (fail).
     """
-    source_paths = luigi.ListParameter(description='List of paths to VCFs to be loaded.')
+
+    source_paths = luigi.Parameter(description='Path or list of paths of VCFs to be loaded.')
     dest_path = luigi.Parameter(description='Path to write the matrix table.')
     genome_version = luigi.Parameter(description='Reference Genome Version (37 or 38)')
     vep_runner = luigi.ChoiceParameter(choices=['VEP', 'DUMMY'], default='VEP', description='Choice of which vep runner'
                                                                                             'to annotate vep.')
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        try:
+            self.source_paths = json.loads(self.source_paths)
+        except json.JSONDecodeError as e:
+            self.source_paths = [self.source_paths]
```

Comment:
I think it's fine since this is what the ListParameter does https://github.com/spotify/luigi/blob/master/luigi/parameter.py#L1054. I'd actually just copy `list(json.loads(x, object_pairs_hook=_FrozenOrderedDict))`

Reply:
I feel pretty strongly that the CLI should stick to conventions whenever possible - especially since --source-paths is the main input parameter and gets changed on every execution.

Reply:
I agree with the general idea, though I would claim that sticking to conventions would mean not overriding the library's CLI argument parsing. But I'm not the one going to be using this, so I won't hold up the PR over this.